dictdiffer/utils.py

```diff
@@ -252,7 +253,7 @@ def dot_lookup(source, lookup, parent=False):
     return value
 
 
-def are_different(first, second, tolerance):
+def are_different(first, second, tolerance, absolute_tolerance):
```

Comment:
Can we add a default value for the `absolute_tolerance` parameter to avoid breaking change?

Reply:
Sure, done.